quartz/build.ts

```diff
@@ -68,12 +73,20 @@ async function buildQuartz(argv: Argv, mut: Mutex, clientRefresh: () => void) {
 
   const parsedFiles = await parseMarkdown(ctx, filePaths)
   const filteredContent = filterContent(ctx, parsedFiles)
+
+  const depGraphs: Record<string, DepGraph> = {}
```

Comment:
we should share one `DepGraph` across all emitters and just have edges and nodes annotated by emitter name